docs/source/ci.rst

```diff
@@ -76,3 +76,35 @@ Specifically for mongo, the :code:`- image: <SERVICE IMAGE>` portion should look
       command: "mongod --journal"
 
 You will receive a `ContainerCheckFailed: Unable to connect to [...] Mongo test container` error in CI if the above is not added to you job config.
+
+
+GitLab
+------
+For :code:`pytest-mock-resources` to work on GitLab use of :code:`dind` service is required.
+Below is a sample configuration:
+
+.. code-block:: yaml
+
+    services:
+      - docker:dind
+
+    variables:
+      DOCKER_TLS_CERTDIR: ''
+
+    stages:
+      - testing
+
+    testing-job:
+      image: python:3.6.8-slim # Use a python version that matches your project
```

Comment:
```suggestion
      image: python:3.6.8-slim  # Use a python version that matches your project
```

This is a minor (unnecessary) change -  @michaelbukachi could you commit this so that CI is triggered? The setting which allows builds on forks was previously turned off.

Reply:
Alright.